app/contributing/style-guide.md

```diff
@@ -2,71 +2,111 @@
+
+| ✅&nbsp; Active       | ❌&nbsp; Passive |
+|---                    | ---               |
+| The plugin *applies* rate limiting to consumers. | Rate limits *are applied to* consumers by the plugin. |
+| You *can explore* the API using a browser. | The API *can be explored* using a browser.  |
+| The YAML file *specifies* the replica count. | The replica count *is specified* in the YAML file.   |
+
+There are exceptions to this guideline.
+You might use passive voice to communicate that the subject is passive and is having something done to it.
+
+For example: "The CA provider's external account can be registered automatically". 
+In this sentence, the account is passive and isn't doing anything, so you can use passive voice.
+
+### Present tense
+
+Whenever possible, use present tense instead of past or future.
+
+| ✅&nbsp; Do use     | ❌&nbsp; Don't use |
+|---                  | ---                |
+|This `command` *starts* a proxy.          | This `command` *will start* a proxy.  |
+|This `command` *starts* a proxy.          | This `command` *has started* a proxy.     |
+
+### Contractions
+
+Don't be afraid to use contractions (*can't*, *isn't*, *you'll*, and so on)! It adds to the conversational tone.
+
+As with everything, there are exceptions to the rule.
+You can leave contractions out when you want a more serious tone, usually to emphasize a warning or caution. 
+
+For example, you might use a contraction in a phrase like "This plugin **isn't** available in Konnect" to let a reader know about plugin availability.
+On the other hand, you might say "**Do not** use this plugin in Konnect because it will break things" to warn them about potential consequences.
+
+### Latin phrases
+
+Don't use Latin phrases or short forms. Use the english version of the same phrase.
+
+| ✅&nbsp; Do use    | ❌&nbsp; Don't use |
+|---                 | ---         |  
+| For example, ...   | e.g., ...   |
+| That is, ...       | i.e., ...   |
+| So (or therefore), ...      | Ergo, ...   |
+
+### Recommendations
 
-## Punctuation rules
+Recommendations should have a reason. If you’re recommending an approach, explain why.
 
-- Commas and periods always go inside quotation marks, and colons and semicolons (dashes as well) go outside.
-  - For example: “There was a storm last night,” Paul said.
+A recommendation should apply to most users. It’s not a suggestion. 
+For example, we wouldn’t recommend DB-less mode without context, but we could recommend it in a specific use case that explains why.
 
-### List punctuation
+When recommending a course of action, use the phrase "we recommend". 
+
+| ✅&nbsp; Do use    | ❌&nbsp; Don't use |
+|---                 | ---                | 
+| **We recommend** using an access token **because it's more secure**. | **Kong recommends** using an access token. |
+| **We don't recommend** using a password in plaintext **because it's not secure**. | **It is recommended** that you use an access token. |
```

Comment:
```suggestion
| **We don't recommend** storing a password in plaintext **because it's not secure**. | **It is recommended** that you use an access token. |
```